apps/req.c

```diff
@@ -868,14 +855,13 @@ int req_main(int argc, char **argv)
 
         i = do_X509_REQ_verify(req, tpubkey, vfyopts);
 
-        if (i < 0) {
+        if (i < 0)
             goto end;
-        } else if (i == 0) {
+        else if (i == 0)
```

Comment:
else after goto is an issue for me, FWIW.

Reply:
Yeah, funny style. Fixed.